pkg/apisix/cluster.go

```diff
@@ -50,6 +51,19 @@ var (
 	ErrDuplicatedCluster = errors.New("duplicated cluster")
 
 	_errReadOnClosedResBody = errors.New("http: read on closed response body")
+
+	// Default shared transport if apisix client
```

Comment:
```suggestion
	// Default shared transport for apisix client
```

We may export options to customize this transport in the future (in command line options).